zlib/grabber.go

```diff
@@ -121,9 +150,6 @@ func makeHTTPGrabber(config *Config, grabData GrabData) func(string) error {
 			tlsConfig.RootCAs = config.RootCAPool
 			tlsConfig.HeartbeatEnabled = true
 			tlsConfig.ClientDSAEnabled = true
-			//		if !config.NoSNI && session.domain != "" {
-			//			tlsConfig.ServerName = session.domain
-			//		}
```

Comment:
I had originally commented out the entire section when first writing the code, but forgot to add it back in. This has already been changed in master and should go away once I rebase.